src/cpu/flexcpu/simple_dataflow_thread.cc

```diff
@@ -406,6 +406,7 @@ SDCPUThread::commitInstruction(std::shared_ptr<InflightInst> inst_ptr)
 void
 SDCPUThread::dumpBuffer()
 {
+#if TRACING_ON
```

Comment:
How about using `if(DTRACE(SDCPUBufferDump))`. I like to avoid the preprocessor directives as much as possible. See http://learning.gem5.org/book/part2/debugging.html#_CPPv26DTRACE4Flag for details